
message StripeInfo
{
    required int32 stripe_version = 1;
    optional uint32 field_type = 2;
    optional uint32 max_dl = 3;
    optional uint32 max_rl = 4;
    optional string root_name = 5;
    optional string root_type = 6;
    repeated uint64 splitting_uid = 7;
    optional uint64 column_uid = 8;
}
